phpcs.xml

```diff
@@ -9,6 +9,10 @@
 	<rule ref="WordPress-Docs"></rule>
 	<rule ref="WordPress-Core"></rule>
 
+	<rule ref="Generic">
+		<exclude name="Generic.Arrays.DisallowShortArraySyntax"/>
```

Comment:
At what point does this go into TribalScents?

Reply:
It is there now! I need to edit this phpcs.xml file. Good catch.

Reply:
🏈